.github/scripts/build.sh

```diff
@@ -64,3 +64,5 @@ echo =====
 ${DOCKER} bash -c "cd build && cpack -G TGZ -D CPACK_PACKAGE_FILE_NAME=psidk-ubuntu-${UBUNTU_BUILD_VER}"
 echo =====
 ${DOCKER} bash -c "cd build && mv book psidk-book && tar czf ../psidk-book.tar.gz psidk-book"
+echo =====
+${DOCKER} bash -c "cargo generate -p ./package-templates/ --destination ./services/user/ --init -v --name Buildtest --silent basic-01 && cd services/user/Buildtest && /root/psibase/build/psidk/bin/psibase package"
```

Comment:
Should be `cargo-psibase`

Reply:
Wouldn't I see that in the contributor image? That's what I was basing this on.
<img width="622" alt="image" src="https://github.com/user-attachments/assets/13e5b190-91b0-47e2-8ff5-4ba35e3680b4" />

Reply:
`psibase` has no `package` command...

Reply:
From the contributor image:
```
$ which cargo-psibase
/root/psibase/build/rust/release/cargo-psibase
```